platform/mellanox/mlnx-platform-api/sonic_platform/component.py

```diff
@@ -60,12 +74,39 @@ def _get_command_result(self, cmdline):
         return result
```

Comment:
@stephenxs do we need also check Exit Code here?

Reply:
It's OK here. 
But it is required in some other place. I added.

Reply:
@stephenxs ok.